client-side/src/components/settings/manageNotifications/EmailFrequency.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import Select from '../../../stories/Select/Select.jsx';
+import GenericButton from '../../../stories/Button/GenericButton.jsx';
+
+const emailFrequencyEnum = {
+  'never':'🚫',
+  'daily':'📅',
+  'weekly':'🗓️',
+  'monthly':'📆',
+  'yearly':'📅'
+};
+
+
+const EmailFrequency = () => {
+  const [emailFrequency, setEmailFrequency] = useState('');
+  const [message, setMessage] = useState('');
+  const preferenceId = '66930c2e2aad987e24078e12';//TO DO: get from state
```

Comment:
I dont see you fixed it, and now I realize that it's also unnecessary in state, you get it from props with all the preferences data.

Reply:
I didn't understand why it needed to be in state and not get it from a props, so I didn't change it